packages/tldraw/src/lib/shapes/shared/TextLabel.tsx

```diff
@@ -26,7 +23,7 @@ type TextLabelProps = {
 	verticalAlign: TLDefaultVerticalAlignStyle
 	wrap?: boolean
 	text: string
-	labelColor: TLDefaultColorStyle
+	labelColor: string
```

Comment:
First change here is superficial: let's change `labelColor` from taking a style prop to taking a regular color instead. We're generally moving in this direction anyway; but here it lets us remove the `isNote` prop that I'd added in a previous PR.